### PR TITLE
เพิ่ม SearchBox

### PR DESCRIPTION
**Pull request by gunhappy**

### CODE REVIEW

**Comment by gunhappy** [comment]
https://github.com/S-O-L-A-R/solarxui/issues/10

**Comment by gunhappy** [comment]
#10

File: src/components/SearchBox/index.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	placeholder: string
+	width: string
+	className?: string
+}
+
+interface ContainerProps {
+	width: string
+}
+
+const Container = styled.div`
```
[the remark this answers is not in the document]
Comment (gunhappy) [reply src/components/SearchBox/index.tsx]: done

File: src/components/SearchBox/index.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		}
+	}
+	> *:not(:last-child) {
+		margin-right: 8px;
+	}
+`
+
+const SearchBox = ({ placeholder, className, width, ...props }: Props) => {
+	return (
+		<Container className={className} width={width}>
```
Comment (ReiiYuki) [inline src/components/SearchBox/index.tsx]: In my opinion I think this should be always responsive (width controller is container wrapper)

Comment (gunhappy) [reply src/components/SearchBox/index.tsx]: fixed

File: src/components/SearchBox/stories.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { text } from '@storybook/addon-knobs'
+import styled from 'styled-components'
+import SearchBox from './'
```
Comment (ReiiYuki) [inline src/components/SearchBox/stories.tsx]: '.'

Comment (gunhappy) [reply src/components/SearchBox/stories.tsx]: เคร autosuggest 555

**Pull request merged**